9.2/run-postgresql-master.sh

```diff
@@ -2,27 +2,16 @@
 
 set -eu
 
-source ${HOME}/common.sh
+source $HOME/common.sh
```

Comment:
not a high prio... i am fixing those when i touching files :)
On Aug 24, 2015 5:14 PM, "Rodolfo Carvalho" <notifications@github.com>
wrote:

> In 9.2/run-postgresql-master.sh
> <https://github.com/openshift/postgresql/pull/51#discussion_r37762118>:
>
> > @@ -2,27 +2,16 @@
> >
> >  set -eu
> >
> > -source ${HOME}/common.sh
> > +source $HOME/common.sh
>
> Thanks for pointing that out. I didn't know we followed anything specific
> like that.
>
> In order of precedence: Stay consistent with what you find; ...; prefer
> "${var}" over "$var"
>
> In multiple places in the same file and group of files we're using a mix
> of both styles. @mfojtik <https://github.com/mfojtik> @bparees
> <https://github.com/bparees> what do you think about a follow up making
> those consistent?!
>
> One way or another is fine for me, as long as we keep ourselves consistent.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/postgresql/pull/51/files#r37762118>.
>